src/hooks/useOrders.ts

```diff
@@ -188,7 +188,7 @@ export function useOrders(): Result {
    */
   const forceOrdersRefresh = useCallback((): void => {
     fetchOrdersAndHandleErrors(userAddress, networkId, 0, areThereOrders(networkId, userAddress, ordersState))
-  }, [fetchOrdersAndHandleErrors, userAddress, networkId, ordersState, areThereOrders])
+  }, [fetchOrdersAndHandleErrors, userAddress, networkId, ordersState])
```

Comment:
Why remove `areThereOrders`?

Reply:
I think my hooks linter did that :scream:

Reply:
getting:
```ts
React Hook useCallback has an unnecessary dependency: 'areThereOrders'. Either exclude it or remove the dependency array. Outer scope values like 'areThereOrders' aren't valid dependencies because mutating them doesn't re-render the component.eslint(react-hooks/exhaustive-deps)
```

Reply:
Nothing against removing it, just wanted to understand why.

BTW, something is not working on my vscode, because the hooks deps are not being updated anymore.

Reply:
try reinstalling the addon?